examples/mediapipe/customMediaPipe/index.html

```diff
@@ -0,0 +1,227 @@
+            display: block;
+        }
+
+        .maincardlayout{
+            display: grid;
+            grid-template-columns: 454px 674px;
+            overflow: hidden;
+        }
+
+        .sourecevideo{
+            aspect-ratio: 1;
+            width: 100%;
+            background: white;
+            border-radius: 4px;
+            height: 100%;
+            object-fit: cover;
+            border: 1px solid var(--vvd-color-neutral-30);
+        }
+        
+        .previewvideodiv{
+            aspect-ratio: 1;
+            width: 100%;
+            background-image: url(images/vonage.png);
+            border-radius: 4px;
+            height: 100%;
+            object-fit: cover;
+            border: 1px solid var(--vvd-color-neutral-30);
+        }
+        .previewvideo{
+            width: 100%;
+            border-radius: 4px;
+            height: 100%;
+            object-fit: cover;
+        }
+	</style>
+
+	<script type="module">
+            import "@vonage/vwc-top-app-bar";
+            import "@vonage/vwc-layout";
+            import "@vonage/vwc-text";
+            import "@vonage/vwc-banner";
+            import "@vonage/vwc-card";
+            import "@vonage/vwc-switch";
+            import "@vonage/vwc-select";
+            import "@vonage/vwc-list/vwc-list-item";
+	</script>
+  <title>MediaPipe sample</title>
+  <link rel="icon" href="https://vonage-client-media-samples.s3.amazonaws.com/vonage.png">
+</head>
+<body>
+<vwc-top-app-bar alternate>
+	<img slot="navigationIcon" height="24" src="https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcQQI9k8NKq3KoOs1VkwSYwPwwQbisiS_K-0jtBUyml13k7EBy8Aekz7G7I1vqceWf5kEQ&usqp=CAU"/>
+	<div>
+        <vwc-banner open connotation="info" message="Vonage MediaPipe integration sample application"></vwc-banner>
+		<vwc-layout gutters="xs" column-basis="block">
+            <vwc-text font-face="title-1" class="headline" tight>This example shows the use of this MediaPipe models:</vwc-text>
+            <vwc-text font-face="subtitle-2" class="headline" tight>Face Detection/Face Mash/Hands/Holistic/Objectron (Cup)/Pose/Selfie Segmentation</vwc-text>
+		</vwc-layout>
+
+        <vwc-card class="maincard" elevation="0">
+            <div slot="main" gutters="xs" class="maincardlayout">
+                <vwc-layout column-basis="block" gutters="xs">
+                    <vwc-text font-face="subtitle-2" class="headline" tight>Source</vwc-text>
+                    <video id="source_video" class="sourecevideo"></video>
+                    <vwc-layout column-spacing="md" style="--layout-grid-template-columns: 1fr 1fr;">
+                        <vwc-text font-face="subtitle-2" tight>Show Results</vwc-text>
+                        <vwc-switch id="cameraswitch" style="justify-self: end;"></vwc-switch>
+                        <vwc-text font-face="subtitle-2" tight>Type</vwc-text>
+                        <vwc-select dense="" id="typeSelector">
+                            <vwc-list-item
```

Comment:
in the vwc-list-item all the properties are not needed. 
You only need this:
<vwc-list-item value="Choose_an_item" selected disabled>Choose MediaPipe module</vwc-list-item>
<vwc-list-item value="Face_Mesh">Face Mesh</vwc-list-item>

Reply:
done